stable/gitlab-ce/values.yaml

```diff
@@ -0,0 +1,89 @@
+## GitLab CE image
+## ref: https://hub.docker.com/r/gitlab/gitlab-ce/tags/
+##
+image: gitlab/gitlab-ce:8.13.3-ce.0
+
+## Specify a imagePullPolicy
+## 'Always' if imageTag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## The URL (with protocol) that your users will use to reach the install.
+## ref: https://docs.gitlab.com/omnibus/settings/configuration.html#configuring-the-external-url-for-gitlab
+##
+#externalUrl: http://your-domain.com/
```

Comment:
We can't un-comment it without risking a bunked install. This and the notice in NOTES.txt along with the README instructions should make sure it gets done.